build.gradle

```diff
@@ -154,6 +158,21 @@ configurations.all {
     System.setProperty("org.gradle.internal.http.socketTimeout", 120000 as String)
 }
 
+task updateGTNHBuildScript {
```

Comment:
do wew want it  to be `GTNH` specific? Besides plugin this is pretty generic

Reply:
Agreed